src/introduction/a-path-to-pos-zcash.md

```diff
@@ -14,35 +14,58 @@ We envision a *proof-of-stake transition path* as one of the potential paths wit
 
 After this transition to pure PoS, there are likely to be future improvements to the PoS protocol, or the consensus protocol more generally. This TFL book focuses almost exclusively on the first step in this sequence.
 
 Our primary motivation for proposing (at least) two steps is to minimize disruption to usability, safety, security, and the ecosystem during each step.
 
+This book primarily focuses this first step: the transition to [PoW+TFL](../terminology.md#definition-pow-tfl). To understand the specific goals for that, see [Design Goals](../overview/design-goals.md). 
+
 With this approach, the Zcash Tech Tree with the [TFL](../terminology.md#definition-tfl) approach might look something like this:
 
 ```dot process
 {{#include ../diagrams/zcash-tech-tree-tfl-steps.dot}}
 ```
 
-## Design Goals for a Hybrid PoW/PoS System
+## Why Two Steps?
+
+One question we've gotten in proposing this approach is why take a two step process with an intermediate [hybrid consensus](../terminology.md#definition-hybrid-consensus) protocol, rather than a single transition directly to a [PoS](../terminology.md#definition-pos) protocol? 
+
+Here's how we think about those trade-offs:
+
+### Considering Single Transition (vs Hybrid Multi-Step)
+
+#### Pros
+- We already understand the current PoW protocol well, and if we transition to an existing proven PoS protocol, then we could skip the complexity of an intermediate hybrid stage.
+- The node implementation might be simpler.
+- Explaining to people what is happening might be simpler. Something like “Zcash has been PoW since it launched, but on $DATE (e.g. at block height X) it will switch to PoS.”
+- Given that the issuance in a given time period is bounded by the supply curve, the full amount that was previously allocated to mining rewards becomes immediately available for staking rewards at the switch-over, rather than having to share this amount between mining and staking during the hybrid stage.
+
+#### Cons
+- If there is any unforeseen show-stopping problem in the new protocol or the transition process, we’d have to react to a network-wide issue.
+- It may be more likely to cause ecosystem disruption; unforeseen differences between PoW and PoS might cause various kinds of snags or papercuts throughout the ecosystem, and these would all pop up around the same time, which may lead to a loss of confidence/retention/adoption or at the very least inconvenience many users for some time.
+- Losing miners: since the transition would be all at once, we may lose some number of miners, who are participants and users in the ecosystem. Miners may leave prior to the transition in order to take care of their own needs. If there is some show-stopper in the transition, one possible short-term mitigation would be to fall back on PoW which is well-known, but if we’ve lost most miners, that may no longer be viable.
```

Comment:
```suggestion
- Losing miners: since the transition would be all at once, we may lose some number of miners, who are participants and users in the ecosystem. Miners may leave prior to the transition in order to take care of their own needs. If there is some show-stopper in the transition, one possible short-term mitigation would be to fall back on PoW which is well known, but if we’ve lost most miners, that may no longer be viable.
```